routing.rst

```diff
@@ -955,6 +955,10 @@ To add support for "param converters" we need SensioFrameworkExtraBundle:
 
     $ composer require sensio/framework-extra-bundle
 
+.. versionadded:: 6.2
+
+    SensioFrameworkExtraBundle was deprecated in Symfony 6.2.
```

Comment:
is this true? I mean this is independent release, what bout instead of version added, mark a note or a caution that the bundle is now global inside symfony core as attributes ?

Reply:
Yes, see https://github.com/sensiolabs/SensioFrameworkExtraBundle/blob/master/README.md
I don't know how the param converter works now. Just thought, adding a quick note is better than nothing :-)